src/common/dom-helpers/dom-base.ts

```diff
@@ -0,0 +1,27 @@
+import {Renderer2, ElementRef} from '@angular/core';
+import {AppError} from 'ng2-qgrid/core/infrastructure';
+
+export class DomBase {
+	name = 'dom-events';
+
+	constructor(private renderer: Renderer2, private elementRef: ElementRef) {
+	}
+
+	protected listen(selector: string | any, event: string, onEvent: Function) {
+		let element = this.selectRootElement(selector);
+
+		this.renderer.listen(element, event, () => {
+			setTimeout(onEvent, 200);
+		});
+	}
+
+	protected selectRootElement(selector: string | any) {
```

Comment:
why `Root`? also when see method like select or find u don't expect exception,
consider to rename it to `getElement(selector)`

Reply:
according to this remark, the method has been renamed to 'getElement' in the commit: [blur directive, dom base class: refactoring](https://github.com/qgrid/ng2/pull/39/commits/bb9b43a5374e93ada17ace5c6f028fd5aa7ac651).